src/Table/tablePropTypes.ts

```diff
@@ -2,7 +2,10 @@ import * as PropTypes from "prop-types";
 
 import sortDir from "./sortDirection";
 
-export const sortDirection = PropTypes.oneOf([sortDir.ASCENDING, sortDir.DESCENDING]);
+export const sortDirection = PropTypes.oneOf<"asc" | "desc">([
```

Comment:
Hmm so if we ever change the `sortDirection` values,

https://github.com/Clever/components/blob/5967a9f8d5f1cfa8d7c7efa1ee739f90ecabd0ae/src/Table/sortDirection.js#L1-L4

we'll need to update them here as well.

Not a huge deal! But is there an easy way to avoid this coupling?

Reply:
This does actually type check the values of `sortDirection`, so if the values are modified then the compiler will throw an error here. It doesn't however catch if new values are added. I'm not too worried in this case since the the values of `sortDirection` are very unlikely to change. There may be a better way to do this though so I'll come back to it.

Reply:
Ah good point about the type-checker catching incompatibilities. In that case, this feels lower priority / non-blocking